README.md

```diff
@@ -37,10 +43,6 @@ turboPy Conda environment
 	- If you just plan to develop a code using the existing turboPy framework: `pip install .` 
```

Comment:
I don't think we need this. If they want to just develop an app, then they should just use `pip install turbopy`
```suggestion
```